physics.py

```diff
@@ -12,7 +12,10 @@
 from wpimath.system.plant import DCMotor, LinearSystemId
 from wpimath.units import kilogram_square_meters
 
+from photonlibpy.simulation import PhotonCameraSim, SimCameraProperties, VisionSystemSim
```

Comment:
nit: this import should be grouped with the other 3rd party imports.
```suggestion
from photonlibpy.simulation import PhotonCameraSim, SimCameraProperties, VisionSystemSim
```